source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md

```diff
@@ -31,12 +31,36 @@ step before it considers the connection string and SDAM specifications. In this
 host names is replaced with a single host name. The format is:
 
 ```
-mongodb+srv://{hostname}.{domainname}/{options}
+mongodb+srv://{subdomain}.{domainname}/{options}
 ```
 
 `{options}` refers to the optional elements from the [Connection String](../connection-string/connection-string-spec.md)
 specification following the `Host Information`. This includes the `Auth database` and `Connection Options`.
 
+For the purposes of this document, `{hostname}` will be divided using the following terminology. If an SRV `{hostname}`
```

Comment:
`{hostname}`... isn't defined above anymore? Do you now mean `{subdomain}`? Or is the whole FQDN now the `{hostname}`? Can this be clarified?

Reply:
For clarification, I've changed it to say: `mongodb+srv://{hostname}/{options} `.  `{hostname}` is the string portions between the protocol and options.